scenarios/scenario_template/cheat_sheet.md

```diff
@@ -0,0 +1,10 @@
+# [SERVICE]_Secrets Walkthrough
```

Comment:
Was `_Secrets` meant to be left in? Not sure if this was carried over from sns_secrets

Reply:
Oh good catch, I missed that one. Just fixed it